app/mailers/user_mailer.rb

```diff
@@ -16,6 +16,11 @@ def password_changed(user)
     mail(to: user.email, subject: t('devise.mailer.password_updated.subject'))
   end
 
+  def contact_request(details)
+    @details = details
+    mail(to: Figaro.env.support_email, subject: t('mailer.contact_request.subject'))
+  end
```

Comment:
Can you write a test for this, please? Code Climate tells me this is not tested.